### ISSUE TITLE
Bitwarden Beta Not Being Updated In Play Store

### ISSUE THREAD

**Issue by uniquePWD** [bitwarden/android#4417]
### Steps To Reproduce

1. Go to the Play Store 
2. Chexk the version of the last update 

### Expected Result

It should be 19528

### Actual Result

It's still on 19422

### Additional Context

Should beta users move to the normal release?

### Build Version

19422

### What server are you connecting to?

US

### Environment Details

Pixel 8 Pro, Android 15 

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by jtodddd** [comment]
Hi there,

Thank you for reporting this! It has been escalated for further investigation. If you have more information that can help us, please add it below.

Thanks!

**Comment by gokulkgm** [comment]
Hello @jtodddd ,

Could the beta version be updated after #4377 is fixed?

I moved to beta 2024.11.1 version because of the mentioned issue.

**Comment by uniquePWD** [comment]
Any news on this?

**Comment by BJReplay** [comment]
Still no updates on the beta.

I have a banking app that won't run if I side-load apps like Bitwarden and enable things like autofill or accessibility, so I'd like to test things like

- https://github.com/bitwarden/android/issues/4308
- https://github.com/bitwarden/android/issues/4239
- https://github.com/bitwarden/android/issues/3866

But can't, unless the beta is updated in the Play Store.

I have side-loaded 2024.12.0 (19597) and #4308 and #4239 both are still not resolved (4308 has a URL for addy.io but not simple-login).

I can't test 3866 because the banking app won't run if the side-loaded beta is installed, and to revert to standard beta, I have to uninstall, then re-enroll in early access via the google forms link, which is a pain.

**Comment by trlef19** [comment]
Neither does release

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (December 02, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔔 **Final Notice - Issue Will Be Closed Soon**

This issue was previously marked as stale and will be automatically closed in **3 days** (December 02, 2025) if no further activity occurs.

If you're still experiencing this issue or believe it should remain open, please comment below to prevent automatic closure.

We appreciate your understanding and contribution to keeping our issue tracker organized! 📋

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
